goosebit/settings/const.py

```diff
@@ -1,8 +1,11 @@
 from pathlib import Path
 
+import platformdirs
```

Comment:
Introducing this dependency just to be removed later is confusing. I think some squashing is in order.

Reply:
Squashed.